data/Viveksati5143/events/2023-06-17-first-in-person-meetup.json

```diff
@@ -0,0 +1,23 @@
+{
+  "isVirtual": false,
```

Comment:
This field is no longer needed, consider removing it, see reference [here](https://linkfree.io/docs/quickstart-json)
![nolongern](https://github.com/EddieHubCommunity/LinkFree/assets/19821445/766faa33-2b03-4eb9-a792-d832ffb175e6)
```suggestion
```